server/sonar-web/src/main/js/apps/organizations/components/OrganizationMembers.js

```diff
@@ -42,6 +49,9 @@ export default class OrganizationMembers extends React.PureComponent {
     if (!this.props.loading && notLoadedYet) {
       this.handleSearchMembers();
     }
+    if (this.props.organizationGroups.length <= 0) {
+      this.props.fetchOrganizationGroups(this.props.organization.key);
```

Comment:
if you not admin of a group, the request must not be done (in any case it fails with 403)